libcloud/compute/drivers/equinixmetal.py

```diff
@@ -746,10 +751,8 @@ def create_volume(
     ):
         """
         Create a new volume.
-
```

Comment:
Can we remove these whitespace changes.  The linters may not like them and we also don't want to make noise in the PR that distracts from the expressed change.

Reply:
I don't think we'll want to make this dependent on #1972 and a post-merge rebase. Let's keep this mergeable without tying it to the other PR.

Reply:
Yes fixed